src/test/java/com/sap/cloudfoundry/client/facade/ITVariable.java

```diff
@@ -0,0 +1,51 @@
+package com.sap.cloudfoundry.client.facade;
+
+import java.util.Set;
+
+public class ITVariable {
+
+    private static final String CF_API_ENV = "CF_API";
+    private static final String USER_EMAIL_ENV = "CF_USER_EMAIL";
+    private static final String USER_PASSWORD_ENV = "CF_USER_PASSWORD";
+    private static final String ORG_ENV = "CF_ORG";
+    private static final String SPACE_ENV = "CF_SPACE";
+    private static final String DOMAIN_NAME_ENV = "DOMAIN_NAME";
```

Comment:
Is this the default domain of CF landscape?

Reply:
No, this is a domain for creation. Do you have idea how to change the name?

Reply:
So, you want to pass as env variables all test values? I was thinking that this will be a constant in the test, but it exist scenario when domain name is already takent in specific landscape. Where will be stored values of these env variables specific for the tests?